hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -850,51 +857,46 @@ private void removeNamespaceOwnershipAndVerify(
     createNamespaceAndVerifyOwnership(
         name, propToCreate, expectedOwnerPostCreate, expectedOwnerTypePostCreate);
 
-    catalog.removeProperties(Namespace.of(name), propToRemove);
+    catalog().removeProperties(Namespace.of(name), propToRemove);
 
-    Database database = metastoreClient.getDatabase(name);
+    Database database = hiveMetastoreSetup.metastoreClient.getDatabase(name);
 
     assertThat(database.getOwnerName()).isEqualTo(expectedOwnerPostRemove);
     assertThat(database.getOwnerType()).isEqualTo(expectedOwnerTypePostRemove);
   }
 
   @Test
-  public void testDropNamespace() throws TException {
+  public void testDropEmptyNamespace() throws TException {
```

Comment:
can we please leave refactorings out of this PR? THis file should only have a minimal set of changes (by adding the setup/teardown stuff(